### DISCUSSION TITLE
How i can get IpPort of SimpleTcpClient in client?

### DISCUSSION THREAD

**Discussion by systemsiteseason** [jchristn/SuperSimpleTcp#76] [General]
i want take IpPort of client in client. Can do that?

**Comment by jchristn** [comment]
Hi @systemsiteseason technically possible but not exposed through the library.  I did a simple test exposing the underlying ```TcpClient.Socket.LocalEndPoint```.  Doing ```.ToString()``` on that returned the following:
![image](https://user-images.githubusercontent.com/1421081/117391894-c9002980-aea5-11eb-85b2-9b969a23fcaa.png)

The other thing to consider is that if there is a network with NAT in between, the IP address the client thinks they are using and the IP address the server sees will not align.  

Best bet is to have some kind of unique identifier that is exchanged between client and server, or, have the server convey this information back to the client.
